web/packages/teleport/src/services/integrations/types.ts

```diff
@@ -985,3 +985,15 @@ export type ExportedIntegrationCaResponse = {
   tls: TlsKey[];
   jwt: JwtKey[];
 };
+
+export type IntegrationDeleteRequest = {
+  name: string;
+  clusterId: string;
+  /**
+   * If true, will delete any associated resources
+   * tied to the integration.
+   *
+   * Not all integration kinds supports resource cleanup.
+   */
+  associatedResources?: boolean;
```

Comment:
I think this should be `deleteAssociatedResources` so it's more obvious what it is doing